collectors/systemd-journal.plugin/README.md

```diff
@@ -378,31 +395,245 @@ nano /etc/systemd/system/systemd-journal-remote.service
 sudo systemctl daemon-reload
 ```
 
-To change the port, copy `/lib/systemd/system/systemd-journal-remote.socket` to `/etc/systemd/system/` and edit it.
-Then do `sudo systemctrl daemon-reload`
+This will make http requests as priority.
+
+2. Reload the daemon configs
+
+```sh
+# reload systemd
+sudo systemctl daemon-reload
+```
+
+3. (OPTIONAL) If you want to change the port, edit the socket file of the `systemd-journal-remote`
+```sh
+# copy the service file
+sudo systemctl edit systemd-journal-remote.socket
+```
+
+and add the following lines into the instructed place, and choose your desired port; save and exit.
+
+
+```sh
+[Socket]
+ListenStream=<DESIRED_PORT>
+```
+    
+
+4. Secure the endpoint from unauthorized access. That depends on your setup (e.g firewall setting, reverse proxies, etc)
```

Comment:
Github is freaking out suggesting with code inside, here consider indenting properly the messages so they fall into the right number in the list, you can copy the raw text from this markdown, for some reason it was breaking when I was trying to suggest


2. Reload the daemon configs

   ```sh
   # reload systemd
   sudo systemctl daemon-reload
   ```

3. (OPTIONAL) If you want to change the port, edit the socket file of the `systemd-journal-remote`

   ```sh
   # copy the service file
   sudo systemctl edit systemd-journal-remote.socket
   ```

   and add the following lines into the instructed place, and choose your desired port; save and exit.

   ```sh
   [Socket]
   ListenStream=<DESIRED_PORT>
   ```

4. Secure the endpoint from unauthorized access. That depends on your setup (e.g firewall setting, reverse proxies, etc)